tests/integration/test_multi.py

```diff
@@ -189,3 +189,41 @@ def test_multi_with_maxmemory(cluster):
     assert node.execute('MULTI') == b'OK'
     assert node.execute('GET', 'key1') == b'QUEUED'
     assert node.execute('EXEC') == [val.encode()]
+
```

Comment:
Should we have "multi with scripts" tests? As far as I can see it should work according to the current code but let's have a test for it.